modules/util/source-formatter/src/main/java/com/liferay/source/formatter/check/JavaUpgradeFDSDataProviderCheck.java

```diff
@@ -0,0 +1,183 @@
+			String fileContent)
+		throws Exception {
+
+		List<String> importNames = javaTerm.getImportNames();
+
+		String javaTermContent = javaTerm.getContent();
+
+		if (!importNames.contains(
+				"com.liferay.frontend.data.set.provider.FDSDataProvider")) {
+
+			return javaTermContent;
+		}
+
+		Matcher implementsFDSDataProviderMatcher =
+			_implementsFDSDataProviderPattern.matcher(javaTermContent);
+
+		if (implementsFDSDataProviderMatcher.find()) {
+			javaTermContent = _sortVariablesMethods(javaTermContent);
+		}
+
+		Matcher variableFDSDataProviderMatcher =
+			_variableFDSDataProviderPattern.matcher(javaTermContent);
+
+		if (variableFDSDataProviderMatcher.find()) {
+			javaTermContent = _sortVariablesMethodCalls(
+				javaTermContent, variableFDSDataProviderMatcher.group(1));
+		}
+
+		return javaTermContent;
+	}
+
+	@Override
+	protected String[] getCheckableJavaTermNames() {
+		return new String[] {JAVA_CLASS};
+	}
+
+	private boolean _hasClassName(
+		String content, String fileContent, String variableName) {
+
+		return Objects.equals(
+			getVariableTypeName(content, fileContent, variableName),
+			"HttpServletRequest");
+	}
+
+	private String _sortVariablesMethodCalls(
+		String content, String variableName) {
+
+		Matcher methodCallGetItemsMatcher = _methodCallGetItemsPattern.matcher(
+			content);
+
+		while (methodCallGetItemsMatcher.find()) {
+			if (variableName.equals(methodCallGetItemsMatcher.group(1)) &&
+				_hasClassName(
+					content, content, methodCallGetItemsMatcher.group(3))) {
+
+				String newOrderVariablesMethodCall = StringBundler.concat(
+					methodCallGetItemsMatcher.group(4),
+					StringPool.COMMA_AND_SPACE,
+					methodCallGetItemsMatcher.group(5),
+					StringPool.COMMA_AND_SPACE,
+					methodCallGetItemsMatcher.group(3),
+					StringPool.COMMA_AND_SPACE,
+					methodCallGetItemsMatcher.group(6));
+
+				String newLine = StringUtil.replace(
+					methodCallGetItemsMatcher.group(0),
+					methodCallGetItemsMatcher.group(2),
+					newOrderVariablesMethodCall);
+
+				content = StringUtil.replace(
+					content, methodCallGetItemsMatcher.group(0), newLine);
+			}
+		}
+
+		Matcher methodCallGetItemsCountMatcher =
+			_methodCallGetItemsCountPattern.matcher(content);
+
+		while (methodCallGetItemsCountMatcher.find()) {
+			if (variableName.equals(methodCallGetItemsCountMatcher.group(1)) &&
+				_hasClassName(
+					content, content,
+					methodCallGetItemsCountMatcher.group(3))) {
+
+				String newOrderVariablesMethodCall = StringBundler.concat(
+					methodCallGetItemsCountMatcher.group(4),
+					StringPool.COMMA_AND_SPACE,
+					methodCallGetItemsCountMatcher.group(3));
+
+				String newLine = StringUtil.replace(
+					methodCallGetItemsCountMatcher.group(0),
+					methodCallGetItemsCountMatcher.group(2),
+					newOrderVariablesMethodCall);
+
+				content = StringUtil.replace(
+					content, methodCallGetItemsCountMatcher.group(0), newLine);
+			}
+		}
+
+		return content;
+	}
```

Comment:
For method calls, you fetch the parameters using `JavaSourceUtil.getParameterList()`.